server/apps/dictionaries/resource.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+
+
+base_dictionary_schema = {
+    'name': {
+        'type': 'string',
+        'unique': True,
+        'required': True
+    },
+    'language_id': {
+        'type': 'string',
+        'required': True
+    },
+    'language_name': {
```

Comment:
Currently we don't have a languages collection containing language metadata; maybe it would have been a good idea to add it now...